packages-common/nanoflow-commons/src/local-storage/GetStorageItemObjectList.ts

```diff
@@ -80,7 +79,7 @@ function GetStorageItemObjectList(key?: string, entity?: string): Promise<mendix
             mx.data.get({
                 guid,
                 callback: mxObject => resolve(mxObject),
-                error: (error: Error) => reject(error)
+                error: (error: Error) => reject(new Error(error.message))
```

Comment:
does or typing not provide type Error?
```suggestion
                error: error => reject(error)
```